vllm/scripts.py

```diff
@@ -0,0 +1,29 @@
+# The CLI entrypoint to vLLM.
+import argparse
+
+from vllm.entrypoints.openai.api_server import run_server
+from vllm.entrypoints.openai.cli_args import make_arg_parser
+
+
+def main():
+    parser = argparse.ArgumentParser(description="vLLM CLI")
+    subparsers = parser.add_subparsers()
+
+    serve_parser = subparsers.add_parser(
+        "serve",
+        help="Start the vLLM OpenAI Compatible API server",
+        usage="vllm serve <model_tag> [options]")
+    make_arg_parser(serve_parser)
+    # Override the `--model` optional argument, make it positional.
+    serve_parser.add_argument("model", type=str, help="The model tag to serve")
```

Comment:
what's happening if vllm serve <model> --model <model2>?

Reply:
renamed the positional arg to be `model-tag` to avoid namespace conflict